usaspending_api/common/helpers/spark_helpers.py

```diff
@@ -534,6 +548,72 @@ def log_hadoop_config(spark: SparkSession, config_key_contains=""):
     ]
 
 
+def load_dict_to_delta_table(spark, s3_data_bucket, table_schema, table_name, data, overwrite=False):
```

Comment:
docstring describing the purpose of this function and when it should/could be used would be helpful.